### DISCUSSION TITLE
Option to disable automatic roboport robot requests

### DISCUSSION THREAD

**Discussion by FluffyDiscord** [notnotmelon/factorissimo-2-notnotmelon#81] [Ideas]
It's annoying, and to work around them you need to manually remove the logistic request. Would be nice to have settings entry that removes this default logistic request.

BTW somehow you need to sometimes remove the logistic request 3-4 times to make it actually stop requesting the bots. A bug?

**Comment by notnotmelon** [comment]
Its added. Horray!

**Reply by FluffyDiscord** [reply]
Thank you! 👍👍👍😊
